lib/main.dart

```diff
@@ -21,13 +22,11 @@ class MyApp extends StatelessWidget {
     routes: <GoRoute>[
       GoRoute(
         path: routePathRootScreen,
-        builder: (BuildContext context, GoRouterState state) =>
-            const HomeScreen(),
+        builder: (BuildContext context, GoRouterState state) => const LoginScreen(),
```

Comment:
@ducbm051291 

We can shortern this code by omit the types.

```suggestion
        builder: (context, state) => const LoginScreen(),
```

Or even omit the parameters' name as we don't use these values:

```suggestion
        builder: (_, __) => const LoginScreen(),
        // change `context` to 1 underscore
        // change `state` to 2 underscores
```